salt/states/nexus.py

```diff
@@ -0,0 +1,147 @@
+# -*- coding: utf-8 -*-
+'''
+This state downloads artifacts from Nexus 3.x.
+
+'''
```

Comment:
A version added tag would be good here as well.

Reply:
Fixed